lib/components/DepositForm.jsx

```diff
@@ -1,21 +1,20 @@
-import React from 'react'
 import { ethers } from 'ethers'
-
 import { Button } from 'lib/components/Button'
 import { FormLockedOverlay } from 'lib/components/FormLockedOverlay'
 import { TextInputGroup } from 'lib/components/TextInputGroup'
 import { displayAmountInEther } from 'lib/utils/displayAmountInEther'
 import { numberWithCommas } from 'lib/utils/numberWithCommas'
+import React from 'react'
 
 export const DepositForm = (props) => {
-  const { genericChainValues, handleSubmit, vars, stateSetters, disabled, usersChainValues } = props
+  const { poolChainValues, handleSubmit, vars, stateSetters, disabled, usersChainValues } = props
```

Comment:
This is great, I was thinking we should rename generic to pool. I originally did generic since it was pool, prize strategy, ticket (controlled tokens), etc. But pool makes sense and is easier to read.